src/App.js

```diff
@@ -1,18 +1,114 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
 
+import { RecipeList } from './components/RecipeList';
+import { SearchBar } from './components/SearchBar';
+
+import { APP_ID, APP_KEY } from './config/';
+
+const ErrorMessage = ({message, isLoading}) => {
+  return (
```

Comment:
I like to write my functions without `return`s if I can get away with it, like here!